config.py

```diff
@@ -0,0 +1,22 @@
+import telebot
+from telebot import types
+
+from datetime import datetime
+from random import randint
+
+import requests
+import json
+from time import sleep
+from DataBase import *
+
+
+#Bot Token
+TOKEN = '5021646895:AAF3VO-BH2jfxnRW03k8b5QzoHbWek1epWo'
```

Comment:
There is a good instrument to work with secret tokens: https://github.com/theskumar/python-dotenv